src/hooks/useCreateMap.ts

```diff
@@ -4,68 +4,61 @@ import * as path from 'path'
 import * as md5 from 'js-md5'
```

Comment:
I think the simplest solution here is to just copy and paste `useCreateMap.ts` from `develop`, assuming you didn't touch this file for this PR